pyproject.toml

```diff
@@ -23,8 +23,8 @@ grpcio = "^1.62.1"
 playwright = "^1.43.0"
 loguru = "^0.7.2"
 markdownify = "^0.12.1"
-npiai-proto = "^0.0.3"
-npiai = "^0.0.3"
+npiai-proto = "0.0.4.dev1"
+npiai = { path = "../client-python"}
```

Comment:
We need to find a way to set up a multi-package monorepo instead of publishing & updating internal dependencies for every minor modification 🤔

Reply:
Also thinking...